modules/pdfchecker/lib/arxiv_pdf_checker.py

```diff
@@ -63,7 +63,7 @@
 
 
 NAME = 'arxiv-pdf-checker'
-ARXIV_VERSION_PATTERN = re.compile(ur'v\d$', re.UNICODE)
+ARXIV_VERSION_PATTERN = re.compile(ur'v\d{1,2}$', re.UNICODE)
```

Comment:
I didn't want to make the regexp too general -- matching any number containing a v somewhere -- in order to accommodate that one bozo who has replaced his paper 117 times on arXiv. it's not actually something INSPIRE should be interested in.